lib/pages/farm/components/MyFarmItem.dart

```diff
@@ -5,7 +5,8 @@ import 'package:flutter/material.dart';
 
 import 'package:ax_dapp/service/Controller/Farms/FarmController.dart';
 import 'package:ax_dapp/pages/farm/components/SingleLogoFarmTitle.dart';
-import 'package:ax_dapp/pages/farm/components/DoubleLogoFarmTitle.dart';
+//TODO: Uncomment when dualStakeDialog is functional
```

Comment:
Not enough to say… one day this will be done… make sure a ticket is created and tag it in the comment